lib/Targets/AIEVecToCpp/TranslateAIEVecToCpp.cpp

```diff
@@ -27,6 +28,7 @@
 #include "mlir/Support/IndentedOstream.h"
 #include "mlir/Support/MathExtras.h"
 #include "llvm/ADT/DenseMap.h"
+#include "llvm/ADT/ScopedHashTable.h"
```

Comment:
Did this one get added by accident or really it was missing?

Reply:
It's used by the code (LN 178, 179, 197, 198, 209, 212). It was being included by one of the headers that did not make use of it.